test/test_helper.rb

```diff
@@ -3,6 +3,8 @@
 
 ENV["RAILS_ENV"] = "test"
 
+require "pathname"
```

Comment:
This isn't needed for our setup, but I assume you ran tests without rake to get the error (via `ruby -I`). In that case, we can keep it 👍